pig-tinder-frontend/src/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react"
+import Footer from './Components/Footer'
+import Header from './Components/Header'
+import PigEdit from './Pages/PigEdit'
+import PigIndex from './Pages/PigIndex'
+import PigNew from './Pages/PigNew'
+import PigShow from './Pages/PigShow'
+import Home from './Pages/Home'
+import NotFound from './Pages/NotFound'
+import "./App.css"
+
+import pigs from "./mockPigs" 
+import { Routes, Route } from 'react-router-dom'
+
+
+
+const App = () => {
+
+
+  console.log(pigs);
```

Comment:
I would recommend removing unused console logs.  Also, consider where you can store pigs that can be used throughout your application.